packages/console/src/pages/SignInExperience/components/Preview.tsx

```diff
@@ -131,6 +132,10 @@ const Preview = ({ signInExperience, className }: Props) => {
     };
   }, [postPreviewMessage]);
 
+  useEffect(() => {
+    postPreviewMessage();
+  }, [customPhrases, postPreviewMessage]);
```

Comment:
what about adding `customPhrases` to `postPreviewMessage`'s dependencies? (line 112)

Reply:
cool, updated.